src/renderer/components/swap/Swap.utils.ts

```diff
@@ -257,3 +257,9 @@ export const minBalanceToSwap = (swapFees: Pick<SwapFees, 'inFee'>): BaseAmount
   // Over-estimate balance by 50%
   return feeToCover.times(1.5)
 }
+
+export const maxAmountToSwapMax1e8 = (assetAmount: BaseAmount, inFeeAmount: BaseAmount): BaseAmount => {
```

Comment:
`assetAmountMax1e8: BaseAmount, feeAmount: BaseAmount`

Reply:
Pls add a comment to this function what we expect as params here. Just to remember it in few months from now.